antarest/study/business/all_optional_meta.py

```diff
@@ -58,4 +60,5 @@ def camel_case_model(model: t.Type[BaseModel]) -> t.Type[BaseModel]:
         field.alias = new_alias
         field.validation_alias = new_alias
         field.serialization_alias = new_alias
+        field.alias_priority = 2
```

Comment:
I prefer defining constants like these as global variables for the module.
This enforces maintainability and gives an explicit documentation for the variable using via the naming.
Example:
`DEFAULT_ALIAS_PRIORITY=2`

Reply:
Indeed! Actually I just removed the line, I left it by mistake, this change is not needed.